### PR TITLE
✨ packaging doc

### PR DESCRIPTION
**Pull request by laurentsimon**
* **Please check if the PR fulfills these requirements**
- [] Tests for the changes have been added (for bug fixes / features)
- [X] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
doc update

* **What is the current behavior?** (You can also link to an open issue here)
lack of doc for Packaging check.

* **What is the new behavior (if this is a feature change)?**
doc for Packaging check.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

File: checks/checks.md [hunk 1]
```diff
@@ -86,6 +86,12 @@ This check tries to determine if the project uses a fuzzing system. It currently
 **Remediation steps**
 - Integrate the project with OSS-Fuzz by following the instructions [here](https://google.github.io/oss-fuzz/).
 
+## Packaging 
+
+This check tries to determine if the project is published as a package that other developers can install/download. The check currently only looks for GitHub packaging workflows. There is plan to add support for package manager hubs in the future.
```
Comment (naveensrinivasan) [inline checks/checks.md:91]: Instead of `There is plan` it would be `There is a plan`

Comment (inferno-chromium) [reply checks/checks.md:91]: +1
also add "other package manager hubs" (add other), maybe can add any example if you can think of (X, Y, etc)

File: checks/checks.yaml [hunk 2]
```diff
@@ -101,6 +101,14 @@ checks:
         To help update your dependencies after pinning them, use tools such as 
         Github's [dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/)
         or [renovate bot](https://github.com/renovatebot/renovate).
+  Packaging:
+    description: >-
+      This check tries to determine if the project is published as a package
+      that other developers can install/download. The check currently only looks for
+      GitHub packaging workflows. There is plan to add support for package manager
+      hubs in the future.
+    remediaiton: >-
```
Comment (inferno-chromium) [inline checks/checks.yaml:110]: typo: this is why it does not show in checks.md

File: checks/checks.yaml [hunk 3]
```diff
@@ -101,6 +101,14 @@ checks:
...
+      Publish your project as a downloadable package.
```
Comment (inferno-chromium) [inline checks/checks.yaml:111]: Any link to github publishing doc, probably https://docs.github.com/en/packages/learn-github-packages/publishing-a-package

File: checks/checks.md [hunk 4]
```diff
@@ -86,6 +86,12 @@ This check tries to determine if the project uses a fuzzing system. It currently
 **Remediation steps**
 - Integrate the project with OSS-Fuzz by following the instructions [here](https://google.github.io/oss-fuzz/).
 
+## Packaging 
+
+This check tries to determine if the project is published as a package that other developers can install/download. The check currently only looks for GitHub packaging workflows. There is plan to add support for package manager hubs in the future. 
+
+**Remediation steps**
+
```
Comment (inferno-chromium) [inline checks/checks.md:94]: Missing text here.

**Review by inferno-chromium — APPROVED** [review]
LGTM with comments (that need addressing)

**Comment by naveensrinivasan** [comment]
We need another PR to address these comments.

**Pull request merged**